llvm/lib/Target/RISCV/MCTargetDesc/RISCVBaseInfo.h

```diff
@@ -171,6 +174,12 @@ static inline bool hasRoundModeOp(uint64_t TSFlags) {
 /// \returns true if this instruction uses vxrm
 static inline bool usesVXRM(uint64_t TSFlags) { return TSFlags & UsesVXRMMask; }
 
+/// \returns true if the result in each element depends on the active elements
```

Comment:
`in each element` may not be true, result of reductions have only one element.

Reply:
Is the scalar result of vcpop/vfirst considered an "element"?

Reply:
I guess, I'm finding this tricky to word. Saying "the result depends on the active elements" seems wrong since the vector result always depends on the active elements.

Maybe it's just easier to say "returns true if the result isn't element-wise"